src/EA.Weee.Web.Tests.Unit/Areas/AatfReturn/Controller/ReturnsSummaryControllerTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿namespace EA.Weee.Web.Tests.Unit.Areas.AatfReturn.Controller
+{
+    using System;
+    using System.Web.Mvc;
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Api.Client;
+    using EA.Weee.Core.AatfReturn;
+    using EA.Weee.Requests.AatfReturn;
+    using EA.Weee.Web.Areas.AatfReturn.Controllers;
+    using EA.Weee.Web.Areas.AatfReturn.ViewModels;
+    using EA.Weee.Web.Constant;
+    using EA.Weee.Web.Controllers.Base;
+    using EA.Weee.Web.Services;
+    using EA.Weee.Web.Services.Caching;
+    using FakeItEasy;
+    using FluentAssertions;
+    using Infrastructure;
+    using Xunit;
+
+    public class ReturnsSummaryControllerTests
+    {
+        private readonly IWeeeClient weeeClient;
+        private readonly CheckYourReturnController controller;
+        private readonly BreadcrumbService breadcrumb;
+        private readonly IMapper mapper;
+
+        public ReturnsSummaryControllerTests()
+        {
+            weeeClient = A.Fake<IWeeeClient>();
+            breadcrumb = A.Fake<BreadcrumbService>();
+            mapper = A.Fake<IMapper>();
+
+            controller = new CheckYourReturnController(() => weeeClient, A.Fake<IWeeeCache>(), breadcrumb, mapper);
```

Comment:
Should this be ReturnsSummaryController?

Reply:
It should be. Corrected now.